### PR TITLE
OWLS-97839 - create-weblogic-credentials.sh and create-rcu-credentials.sh should prompt for password values if not passed as argument

### PR DESCRIPTION
**Pull request by alai8**
Currently , create-weblogic-credentials.sh and create-rcu-credentials.sh allows only clear text password as argument.
User should be prompted if password value not passed as argument and value should be masked.

jenkins: https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9707/

### CODE REVIEW

File: kubernetes/samples/scripts/create-rcu-credentials/README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ The parameters are as follows:
 
 ```  
   -u username for schema owner (regular user), must be specified.
-  -p password for schema owner (regular user), must be specified.
+  -p password for schema owner (regular user), must be provided through -p argument or user will be prompted to enter a value.
```
Comment (rosemarymarano) [inline kubernetes/samples/scripts/create-rcu-credentials/README.md:26]: provided through -> provided using the ("using" is the better verb to use in this phrase; please make this correction globally. Thank you.)

File: documentation/staging/content/samples/credentials/_index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The parameters are as follows:
 
 ```  
   -u user name, must be specified.
-  -p password, must be specified.
+  -p password, must be provided using -p argument or user will be prompted to enter a value.
```
Comment (rosemarymarano) [inline documentation/staging/content/samples/credentials/_index.md:27]: provided using -> provided using the

**Review by rosemarymarano — COMMENTED** [review]
minor edits

File: documentation/staging/content/samples/credentials/_index.md (hunk 2 above)
Comment (alai8) [reply documentation/staging/content/samples/credentials/_index.md]: Thanks Rosemary. Updated files with the suggested edits.

**Review by xiancao — APPROVED** [review]
lgtm

**Pull request merged**